tests/unit/ClientTest.php

```diff
@@ -174,9 +182,9 @@ protected function doTestResponse($responseFile, array $result, $method)
      *
      * @return Client
      */
-    protected function getClient($username = '', $password = '',  $ssl = false, $timeout = 0)
```

Comment:
This is a BC Break, I can not merge this PR.
Can you swap the parameters? 

```php
$username = '', $password = '',  $ssl = false, $timeout = 0, $verifySSL = false
```
In this way can work :) 

THanks a lot!